src/main/java/uk/gov/mca/beacons/api/gateways/NoteGatewayImpl.java

```diff
@@ -11,15 +13,19 @@
 @Transactional
 public class NoteGatewayImpl implements NoteGateway {
 
+  private Clock clock;
```

Comment:
could/should we move this in to NoteMapper, make NoteMapper methods non-static, and autowire the clock on the instance?